tests/Functional/ContentRenderTestBase.php

```diff
@@ -8,6 +8,7 @@
 use Drupal\oe_content_entity_contact\Entity\ContactInterface;
 use Drupal\Core\Entity\EntityStorageInterface;
 use Drupal\media\MediaInterface;
+use Drupal\oe_content_entity_document_reference\Entity\DocumentReference;
```

Comment:
Document reference is in modules/oe_content_sub_entity/modules/oe_content_sub_entity_document_reference now.